src/renderer/components/inputs/StaticValueInput.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Center, HStack, Text } from '@chakra-ui/react';
+import { memo, useEffect } from 'react';
+import { useContext } from 'use-context-selector';
+import { ExecutionContext } from '../../contexts/ExecutionContext';
+import { InputContext } from '../../contexts/InputContext';
+import { TypeTags } from '../TypeTag';
+import { WithoutLabel } from './InputContainer';
+import { InputProps } from './props';
+
+export const StaticValueInput = memo(
+    ({ value, setValue, input, definitionType }: InputProps<'static', number | string>) => {
+        const { valueOf } = input;
+        const { executionNumber } = useContext(ExecutionContext);
+        const { conditionallyInactive } = useContext(InputContext);
+
+        useEffect(() => {
+            switch (valueOf) {
+                case 'execution_number':
+                    setValue(executionNumber);
+                    break;
+                default:
+                    setValue(0);
+                    break;
```

Comment:
To ensure that we don't forget anything.

```suggestion
                    assertNever(valueOf);
```